react/components/SKUSelector/utils/index.js

```diff
@@ -0,0 +1,88 @@
+import { clone } from 'ramda'
+
+/**
+ * Return the maximum sku price
+ * @param {array of sku's} items 
+ */
+export const getMaxSkuPrice = items => {
```

Comment:
It is not better to use an `Array.prototype.reduce` method to get the max sku price instead of a `forEach`? See [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) for more details.

Reply:
👍